scripts/us_census/pep/Population_Estimate_by_ASR/national_2000_2010.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This Python Script is
+for National Level Data
+2000-2010.
+'''
+import os
+import pandas as pd
+from common_functions import _input_url
+
+
+def national2000():
+    '''
+    This Python Script Loads csv datasets
+    from 2000-2010 on a National Level,
+    cleans it and create a cleaned csv.
+    '''
+    # Getting input URL from the JSON file.
+    _url = _input_url("national.json", "2000-10")
+    # Reading the csv format input file and converting it to a dataframe.
+    df = pd.read_csv(_url, encoding='ISO-8859-1', low_memory=False)
+    # Removing the unwanted rows.
+    df = df.query("MONTH != 4")
+    df = df.query("YEAR != 2010")
+    df = df.query("AGE != 999")
+    df['AGE'] = df['AGE'].astype(str)
+    df['AGE'] = df['AGE'] + 'Years'
+    df['AGE'] = df['AGE'].str.replace("85Years", "85OrMoreYears")
+    # Dropping unwanted columns.
+    df = df.drop(columns=[
+        'TOT_POP', 'NH_MALE', 'NH_FEMALE', 'NHWA_MALE', 'MONTH', 'NHWA_FEMALE',
```

Comment:
I see this elsewhere too. Could we not have a superset of column names to drop? Then we could use that list everywhere? Even if a df does not have a column, dropping it should be ok, right?

Reply:
When we try to drop a column that does not exist an error is thrown. However have reused the column list where ever possible